packages/react-components/react-card/src/components/Card/useCard.ts

```diff
@@ -26,19 +40,48 @@ export const useCard_unstable = (props: CardProps, ref: React.Ref<HTMLElement>):
     tabBehavior: focusMap[focusMode],
   });
 
-  const focusAttrs = focusMode !== 'off' ? { tabIndex: 0, ...groupperAttrs } : null;
+  const focusAttrs = focusMode !== 'off' ? { ...groupperAttrs } : null;
+
+  const onChangeHandler = (event: React.MouseEvent | React.KeyboardEvent) => {
+    setChecked(!checked);
+    onCardSelect && onCardSelect(event, { selected: checked });
+  };
+
+  const selectAttrs =
+    selectable === true
+      ? {
+          onClick: onChangeHandler,
+          onKeyDown: (event: React.KeyboardEvent<HTMLDivElement>) => {
+            if (event.key === Enter || event.key === Space) {
+              event.preventDefault();
```

Comment:
Not a fan at all of preventing browser defaults but if we don't, it will collide with our interception of `Spacebar` usage. Open to more elegant solutions.

Reply:
I checked the demos and run into interesting situation. is this supposed to work like following ?

![image](https://user-images.githubusercontent.com/1223799/172330052-028f553d-e2e5-4576-91ad-7a257cc39988.png)
![image](https://user-images.githubusercontent.com/1223799/172330263-528d0fd3-91c1-46ce-8e37-fed1464d14dc.png)

Reply:
Under the focus of a checkbox, as you mentioned, Spacebar already checks it. This will check the checkbox, but the event is then bubbled up to the Card handler and it gets checked again.
I'll need some time to analyze this and find out a better approach.

Reply:
I'm also now thinking about how this will interact with the requested focus behaviors, given that Enter is supposed to "go into the card" and focus the inner elements. I guess we need to bring this up with design and clarify.

Reply:
yeah, to me that behaviour is kinda confusing as both events will trigger (selectable and action click) which also ads some lag (freezes UI for second or so - might be Storybook related though). I'd expect to not mark it as selected if I click on action control